### DISCUSSION TITLE
Generate syntax highlighting file for CodeMirror

### DISCUSSION THREAD

**Discussion by vanillajonathan** [pest-parser/pest#801] [Ideas]
Generate syntax highlighting file for the [CodeMirror](https://codemirror.net/) JavaScript editor.

**Comment by tomtau** [comment]
There's a simple mode defined on the website: https://github.com/pest-parser/site/blob/master/static/index.html#L373

**Reply by vanillajonathan** [reply]
Maybe there should be an attempt to upstream that? But maybe it is not popular enough.

Either way, I didn't mean a syntax highlight file for the Pest grammar itself, I meant that Pest should generate a syntax highlight file for CodeMirror from `.pest` files.
